package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "nhost-js-root",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Nhost Javascript SDK",
+  "author": "Nhost",
+  "license": "MIT",
+  "homepage": "https://nhost.io",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nhost/nhost-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/nhost/nhost-js/issues"
+  },
```

Comment:
Do we need to update the name, description, and URLs for this package.json?

Reply:
I'm not fond on this `-js` suffix to be honest. Any npm package is meant to be Javascript, this suffix is redundant. I did not want to change the package name as we are not talking about renaming our existing npm packages, although I think we should

Reply:
Yep I see your point and I agree.

The question is if we want to introduce the mismatch now? Or keep things in sync for now until we make such name change?

Reply:
I haven't changed the package name and can keep add `-js` suffixes to the folders although it is hurting my eyes :)

Reply:
I don't have a particular hard opinion here tbh. Maybe @nunopato wants to chime in?

Reply:
> I haven't changed the package name and can keep add `-js` suffixes to the folders although it is hurting my eyes :)

At least we're being consistent! :D

Reply:
ah ok, I saw your second comment about the root package.json which is somehow incorrect. Pushing changes